Trappist/src/Promact.Trappist.Test/Profile/ProfileRepositoryTest.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Promact.Trappist.DomainModel.DbContext;
+using Promact.Trappist.Repository.Profile;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+using Microsoft.AspNetCore.Identity;
+using Promact.Trappist.Web.Models;
+
+namespace Promact.Trappist.Test.Profile
+{
+    [Collection("Register Dependency")]
+    public class ProfileRepositoryTest
+    {
+        private readonly Bootstrap _bootstrap;
+        private readonly TrappistDbContext _dbContext;
+        private readonly IProfileRepository _profileRepository;
+        private readonly UserManager<ApplicationUser> _userManager;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the "_userManager" field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1450)